sdk/edgezones/arm-edgezones/src/models/models.ts

```diff
@@ -184,7 +184,7 @@ export interface Operation {
   /** Whether the operation applies to data-plane. This is "true" for data-plane operations and "false" for Azure Resource Manager/control-plane operations. */
   readonly isDataAction?: boolean;
   /** Localized display information for this particular operation. */
-  display?: OperationDisplay;
+  readonly display?: OperationDisplay;
```

Comment:
do you know why readonly is imported? could you find the tsp spec link for me?

Reply:
this is from typesprc-azure-resource-manager